unicode-transforms.cabal

```
@@ -172,11 +177,15 @@ benchmark bench
       base >=4.7 && <5
     , deepseq >=1.1.0 && <1.5
     , filepath >=1.0 && <2
-    , gauge >=0.2.0 && <0.3
     , path >=0.0.0 && <0.9
     , path-io >=0.1.0 && <1.7
     , text >=1.1.1 && <1.3
     , unicode-transforms
+  if flag(use-gauge)
+    build-depends: gauge >=0.2.0 && <0.3
+  else
+    build-depends: tasty-bench>= 0.2.5 && <0.3
```

```suggestion
    build-depends: tasty-bench>= 0.2.5 && <0.4
```
`tasty-bench-0.3` is already out